Chapter9/Chapter9/RestaurantTableViewController.swift

```diff
@@ -38,14 +39,36 @@ class RestaurantTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let optionMenu = UIAlertController(title: nil, message: "What do you want to do?", preferredStyle: .actionSheet)
+        guard let selectedCell = tableView.cellForRow(at: indexPath) else { return }
 
-        if let popoverController = optionMenu.popoverPresentationController {
-            if let cell = tableView.cellForRow(at: indexPath) {
-                popoverController.sourceView = cell
-                popoverController.sourceRect = cell.bounds
+        var actionButtonTitle = "Check In"
```

Comment:
Declare at the start of the class. Make more sense and you can declare it as an empty string because you change it in the next lines.